### ISSUE TITLE
Panic when miter limit is too low

### ISSUE THREAD

**Issue by Dinnerbone** [ruffle-rs/ruffle#1194]
In the game [Match-3 Puzzle](https://www.newgrounds.com/portal/view/531068?emulate=flash), we panic when converting the shapes through lyon:

```
thread 'main' panicked at 'assertion failed: limit >= Self::MINIMUM_MITER_LIMIT', C:\Users\dinne\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\src\libstd\macros.rs:13:23
stack backtrace:
   0: backtrace::backtrace::trace_unsynchronized
             at C:\Users\VssAdministrator\.cargo\registry\src\github.com-1ecc6299db9ec823\backtrace-0.3.46\src\backtrace\mod.rs:66
   1: std::sys_common::backtrace::_print_fmt
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\sys_common\backtrace.rs:78
   2: std::sys_common::backtrace::_print::{{impl}}::fmt
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\sys_common\backtrace.rs:59
   3: core::fmt::write
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libcore\fmt\mod.rs:1069
   4: std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\io\mod.rs:1504
   5: std::sys_common::backtrace::_print
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\sys_common\backtrace.rs:62
   6: std::sys_common::backtrace::print
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\sys_common\backtrace.rs:49
   7: std::panicking::default_hook::{{closure}}
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\panicking.rs:198
   8: std::panicking::default_hook
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\panicking.rs:218
   9: std::panicking::rust_panic_with_hook
             at /rustc/49cae55760da0a43428eba73abcb659bb70cf2e4\/src\libstd\panicking.rs:511
  10: std::panicking::begin_panic<str*>
             at C:\Users\dinne\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libstd\panicking.rs:438
  11: lyon_tessellation::StrokeOptions::with_miter_limit
             at C:\Users\dinne\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libstd\macros.rs:13
  12: ruffle_render_wgpu::WgpuRenderBackend<ruffle_render_wgpu::target::SwapChainTarget>::register_shape_internal<ruffle_render_wgpu::target::SwapChainTarget>
             at W:\personal\ruffle\render\wgpu\src\lib.rs:596
  13: ruffle_render_wgpu::{{impl}}::register_shape<ruffle_render_wgpu::target::SwapChainTarget>
             at W:\personal\ruffle\render\wgpu\src\lib.rs:869
  14: ruffle_core::display_object::graphic::Graphic::from_swf_tag
             at W:\personal\ruffle\core\src\display_object\graphic.rs:22
  15: ruffle_core::display_object::movie_clip::MovieClipData::define_shape
             at W:\personal\ruffle\core\src\display_object\movie_clip.rs:1655
  16: ruffle_core::display_object::movie_clip::{{impl}}::preload::{{closure}}
             at W:\personal\ruffle\core\src\display_object\movie_clip.rs:243
  17: ruffle_core::tag_utils::decode_tags<slice<u8>*,closure-0>
             at W:\personal\ruffle\core\src\tag_utils.rs:285
  18: ruffle_core::display_object::movie_clip::MovieClip::preload
             at W:\personal\ruffle\core\src\display_object\movie_clip.rs:351
  19: ruffle_core::player::{{impl}}::preload::{{closure}}
             at W:\personal\ruffle\core\src\player.rs:762
  20: ruffle_core::player::{{impl}}::mutate_with_update_context::{{closure}}<closure-0,()>
             at W:\personal\ruffle\core\src\player.rs:1115
  21: ruffle_core::player::GcArena::mutate<closure-0,()>
             at C:\Users\dinne\.cargo\registry\src\github.com-1ecc6299db9ec823\gc-arena-0.2.0\src\arena.rs:161
  22: ruffle_core::player::Player::mutate_with_update_context<closure-0,()>
             at W:\personal\ruffle\core\src\player.rs:1062
  23: ruffle_core::player::Player::preload
             at W:\personal\ruffle\core\src\player.rs:759
  24: ruffle_core::player::Player::set_root_movie
             at W:\personal\ruffle\core\src\player.rs:382
  25: ruffle_desktop::run_player
             at .\src\main.rs:181
  26: ruffle_desktop::main
```

**Comment by Herschel** [comment]
Seems like this no longer occurs in lyon 0.16.2.

**Comment by Dinnerbone** [comment]
Still happens in movies such as https://www.newgrounds.com/portal/view/537015?emulate=flash (on wgpu backend at least, but I imagine that shouldn't matter)

**Comment by Herschel** [comment]
Fixed by #1611
